my-app/src/components/buttons.tsx

```diff
@@ -1,10 +1,23 @@
 import * as React from "react";
 import "./buttons.css";
+import loadingIcon from "../images/ic_loading.svg";
 
-export class PrimaryButton extends React.Component{
+interface Props{
```

Comment:
```suggestion
interface PrimaryButtonProps {
```
Convenção de nomenclaruta de Props e State é:  `NomeDoComponenteProps` e `NomeDoComponenteState`